tests/basic/pom.xml

```diff
@@ -1,104 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-  ~ Copyright 2015-2016 Red Hat, Inc, and individual contributors.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~ http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <parent>
-        <groupId>org.jboss.hal</groupId>
-        <artifactId>hal-testsuite-tests</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
-    </parent>
-
-    <artifactId>hal-testsuite-tests-basic</artifactId>
-    <name>HAL Testsuite :: Tests :: Basic</name>
```

Comment:
What about keeping name convention in new test modules?

Old modules:
```
[INFO] HAL Testsuite :: Tests ............................. SUCCESS [  0.042 s]
[INFO] HAL Testsuite :: Tests :: RBAC ..................... SUCCESS [  0.056 s]
[INFO] HAL Testsuite :: Tests :: Transaction .............. SUCCESS [  0.059 s]
```

New modules:
```
[INFO] tests-configuration-modcluster ..................... SUCCESS [  0.299 s]
[INFO] tests-configuration-paths .......................... SUCCESS [  0.247 s]
[INFO] tests-configuration-remoting ....................... SUCCESS [  0.366 s]
```

Reply:
I'm not fan of that. With few modules it was IMHO nice to have fancy names, but with a lot of modules you want to find it as fast as possible in case of issues and having module ID same as modules name makes it so.

Reply:
+1, in that case what about to have unique approach to all modules and remove old names from old modules?

Reply:
Makes sense, will remove fancy names.